### DISCUSSION TITLE
MDF AxisDistribution plot

### DISCUSSION THREAD

**Discussion by JOchoaA** [mtex-toolbox/mtex#1860] [Ask Anything] [answered]
Hello, dear community,

I am plotting the MDF axis distribution for r a list of orientations and getting the attached picture. However, I would like to make the poles' font size bigger and have an additional plot that shows individual point dispersion rather than a contour plot (see example). Could someone please help me?

Thanks in advance for any help.

My plot
![p77-alamel_alpha_MDF-axis](https://github.com/mtex-toolbox/mtex/assets/61281507/c7653605-cf18-487c-9a27-5391dac8a3dc)

example

![image](https://github.com/mtex-toolbox/mtex/assets/61281507/88da5147-460b-4bc0-b2b8-dd17eca6d300)

my script

....
odf = calcDensity(oris,'weights',vf,'halfwidth',gs)
mdf = calcMDF(odf);
plotAxisDistribution(mdf,'order',22,'linewidth',0.5,'contourf',levels,'LineStyle','-')

**Comment by kilir** [comment, marked as answer]
Hi,
how about:
```
cs = crystalSymmetry('422');
% simulate an mdf
mdf = calcDensity(orientation.rand(1,cs,cs));

plotAxisDistribution(mdf,'contourf')

p = getappdata(gca,'sphericalPlot')
for i=1:3
p.hgt.Children(i).FontSize = 36
end

nextAxis
plotAxisDistribution(discreteSample(mdf,1000))
```
Cheers,
Rüdiger

**Comment by JOchoaA** [comment]
Thanks a lot Rüdiger.

Have a nice day.

Cheers.

Jhon Ochoa
